docs/pages/dashboard/drop.mdx

```diff
@@ -16,20 +16,22 @@ The Liteflow Infrastructure requires the smart contract to implement at least a
 
```

Comment:
@antho1404 there are no menu-links to the dashboard pages anymore. Are we going to wait for https://github.com/liteflow-labs/liteflow-js/pull/169 to be merged to show these pages?

Reply:
Somehow your build didn't work with the cache or something like that, I re-executed the build and no more issue